cdc/cdc.cc

```diff
@@ -279,7 +279,60 @@ class transformer final {
         auto shard_id = dht::murmur3_partitioner::shard_of(
                 t, locator::i_endpoint_snitch::get_local_snitch_ptr()->get_shard_count(eps[0]), _ignore_msb_bits);
         mutation res(_log_schema, stream_id(*_log_schema, eps[0].addr(), shard_id));
-        set_pk_columns(m.key(), 0, res);
+        auto& p = m.partition();
+        if(p.partition_tombstone()) {
+            // Partition deletion
+            set_pk_columns(m.key(), 0, res);
+        } else if (!p.row_tombstones().empty()) {
```

Comment:
We need to remember that when batch operations are supported, this case and the next will not necessarily be disjoint.